MariiaNovitska/HW5_loops/task_3.py

```diff
@@ -0,0 +1,12 @@
+number = int(input("Write your number to calculate factorial: "))
```

Comment:
good job
look at this code: 


number = int(input("Write your number to calculate factorial: "))

if number < 0:
    print("It's impossible to calculate the factorial for negative numbers.")
else:
    factorial = 1
    for f in range(1, number + 1):
        factorial *= f
    print(f"{number}! = {factorial}")

Reply:
Got it)